doc/rst/meta/modules/standard.rst

```diff
@@ -6,19 +6,120 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+
+Automatic Modules
+-----------------
+
 All Chapel programs automatically ``use`` the modules :chpl:mod:`Builtins` ,
 :chpl:mod:`Math` , and :chpl:mod:`Types` by default.
```

Comment:
We could drop the module names in this statement, as we already list them right after.

Reply:
Good point.  My latest commit does that if you want to take a look.

Reply:
Looks good, thanks!